File: Freestyle/Android/Freestyle/src/freestyle/data/dto/todb/RhymeWithSenToSave.java
package freestyle.data.dto.todb;

public class RhymeWithSenToSave extends RhymeToSave {
	public RhymeWithSenToSave(RhymeToSave rhymeToSave){
		this.replyId = rhymeToSave.replyId;
		this.rhymeId = rhymeToSave.rhymeId;
		this.userKey = rhymeToSave.userKey;
	}
	public RhymeWithSenToSave(){}
	
	public String sentence;
}

File: Freestyle/Webservice/WcfService/Models/DTO/ToDb/RhymeWithSenToSave.cs
namespace WcfService.Models.DTO.ToDb
{
    using System.Runtime.Serialization;

    [DataContract]
    public class RhymeWithSenToSave : RhymeToSave
    {
        [DataMember]
        public string Sentence { get; set; }
    }
}